aiohttp/client_reqrep.py

```diff
@@ -307,6 +328,35 @@ def update_proxy(self, proxy, proxy_auth, proxy_from_env):
         self.proxy = proxy
         self.proxy_auth = proxy_auth
 
+    def update_fingerprint(self, fingerprint):
+        if fingerprint:
+            digestlen = len(fingerprint)
+            hashfunc = HASHFUNC_BY_DIGESTLEN.get(digestlen)
+            if not hashfunc:
+                raise ValueError('fingerprint has invalid length')
+            elif hashfunc is md5 or hashfunc is sha1:
+                warnings.simplefilter('always')
```

Comment:
Wouldn't this cause suddenly global side effect?

Reply:
I have no idea. I copy-pasted that from there a610cf12209326fba5e6660c1644cbff55b7c07d

Reply:
I think you're right. After this the "warning filter specifications" is altered. https://docs.python.org/3.5/library/warnings.html#warnings.simplefilter @asvetlov can you confirm you want to keep this behavior?

Reply:
No.
@kxepal good catch.
Please don't alter `warnings` filter but duplicate the message to `aiohttp.log.client_logger`.
From my experience people are much more sensible to log messages than deprecation warnings.
Log level should be [`warning'](https://docs.python.org/3/library/logging.html#logging.Logger.warning) sure.

Reply:
I suppose I change the one in connector too? ^_^